tests.py

```diff
@@ -482,6 +485,7 @@ def podman_create(tags):
                 'start_time': start_time,
                 'end_time': end_time,
                 'failures': failures,
+                'successes': successes,
```

Comment:
consider names like `success_count` and `failure_count` to avoid confusion

Reply:
Sure. Updated as suggested.